src/Pages/2DPrinting/2DPrinting.js

```diff
@@ -327,39 +328,20 @@ export default function Printing (props) {
             </Button>
           </ModalFooter>
 
-          <Modal
-            isOpen={confirmModal}
+          <ConfirmationModal
+            headerText='Are you sure you want to print?'
```

Comment:
@evanugarte @kalyanpullela 
Are these good with the way they are?
or should we make another object/dictionary for these fields?
Ex:
```js
let confirmModalMsg= {
headerText: 'Are you sure you want to print?',
bodyText: 'Click Yes or Go Back',
...
}
```

and call it in the modal:
```js
<ConfirmationModal
headerText= confirmModalMsg.headerText
bodyText= confirmModalMsg.bodyText
.../>
```

Reply:
same goes for the other 2 files

Reply:
Ahhh good idea. You can also do

```jsx
const confrimModalProps = { /* stufff */ }


<ConfirmationModal
  {...confirmModalProps}
```